foreman.te

```diff
@@ -395,21 +382,18 @@ read_files_pattern(websockify_t, puppet_var_lib_t, puppet_var_lib_t)
 
 ######################################
 #
-# Docker
+# Container / Docker
 #
 
-type docker_port_t;
-corenet_port(docker_port_t)
-
-tunable_policy(`passenger_can_connect_docker_tcp',`
-    allow passenger_t docker_port_t:tcp_socket name_connect;
+optional_policy(`
+    tunable_policy(`passenger_can_connect_docker_tcp',`
```

Comment:
@lzap this seems like a ` and `'` at the end of line are swapped

Reply:
Sorry what? :)